DEVELOPER_GUIDE.md

```diff
@@ -48,4 +48,12 @@ curl -XPOST -u 'admin:admin' 'http://localhost:5601/api/sample_data/ecommerce' -
 curl -XPOST -u 'admin:admin' 'http://localhost:5601/api/sample_data/logs' -H 'osd-xsrf:true'  -H 'securitytenant: global'
 curl -XPOST -u 'admin:admin' 'http://localhost:5601/api/sample_data/flights' -H 'osd-xsrf:true'  -H 'securitytenant: admin_tenant'
 ```
-Run `yarn test` inside `reporting-cli`.
\ No newline at end of file
+Run `yarn test` inside `reporting-cli`.
+
+### Creating Lambda container image
+
+1. Download the opensearch-reporting-cli-1.0.0.tgz file and get [Dockerfile](./Dockerfile).
```

Comment:
i thought the tgz built is `opensearch-project-reporting-cli-1.0.0.tgz`?

Reply:
why build then rename just to remove `-project`? my concern is it might not be clear what the final artifact name would be since this logic is only in CI

Reply:
Ohh the logic is not only in CI. the artifact on downloads page will not have `-project`. We added a step to rename in the build pipeline.

Reply:
sorry i still didn't understand.. why not name the build artifact `opensearch-reporting-cli-1.0.0.tgz` so we don't have to rename in CI and build pipeline?

Reply:
`npm pack` creates `<name>-<version>.tgz`. Couldn't find option to edit this name so I added step to rename. https://docs.npmjs.com/cli/v7/commands/npm-pack. Do you know how?

We can change the [name](https://github.com/opensearch-project/reporting-cli/blob/main/package.json#L2) in package.json, but we kept name `@opensearch-project/reporting-cli` because it follows same naming convention as other opensearch packages on npm

Reply:
Got it, that makes sense. thanks for the explanation